src/pyramid_basemodel/__init__.py

```diff
@@ -55,10 +59,10 @@ class and ``bind_engine`` function.
 class classproperty:
```

Comment:
it's problematic to type it properly. Maybe it could be rewritten from the properties actually...